examples/3-twitter-clone/src/app/models/MessageModel.ts

```diff
@@ -11,12 +8,19 @@ export {
   MessageModelSelector
 } from "./MessageModel.base"
 
+/* The TypeScript type of an instance of MessageModelBase */
+export interface MessageModelType extends Instance<typeof MessageModel.Type> {}
+export interface MessageModelType extends MessageModelBaseRefsType {}
+
+/* Helper function to cast self argument to a MessageModel instance */
+const as = (self: any) => (self as unknown) as MessageModelType
+
 /**
  * MessageModel
  */
 export const MessageModel = MessageModelBase.views(self => ({
   get isLikedByMe() {
-    return self.likes.includes(self.store.me)
+    return as(self).likes.includes(self.store.me)
```

Comment:
I'm getting `Property 'likes' does not exist on type 'MessageModelType'.` here. Should this not work when using `as()`?